### PR TITLE
Add rename script

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -19,4 +19,4 @@ ingress:
 env:
   HMPPS_AUTH_URL: https://sign-in-dev.hmpps.service.justice.gov.uk/auth
   TOKEN_VERIFICATION_API_URL: https://token-verification-api-dev.prison.service.justice.gov.uk
-  TOKEN_VERIFICATION_ENABLED: true
\ No newline at end of file
+  TOKEN_VERIFICATION_ENABLED: true
```
Comment (psoleckimoj) [inline helm_deploy/values-dev.yaml:22]: Newline

Comment (psoleckimoj) [reply helm_deploy/values-dev.yaml:22]: Why can't I delete this comment 😆

File: rename-project.bash [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+echo "Performing directory renames"
+
+# move helm stuff to new name
+mv "helm_deploy/hmpps-template-typescript" "helm_deploy/$PROJECT_NAME"
+
+# lastly remove ourselves
+rm rename-project.bash
+
+echo "Completed."
+echo "Please now review changes"
```
Comment (psoleckimoj) [inline rename-project.bash:48]: Newline....well this one is right 😆

File: server/app.ts [hunk 3]
```diff
@@ -140,6 +140,7 @@ export default function createApp(userService: UserService): express.Application
 
   // GovUK Template Configuration
   app.locals.asset_path = '/assets/'
+  app.locals.applicationName = 'HMPPS Template Typescript'
```
Comment (psoleckimoj) [inline server/app.ts:143]: `HMPPS Typescript Template` sounds better?

Comment (andrewrlee) [reply server/app.ts]: done that

File: rename-project.bash (hunk 2 above)
Comment (andrewrlee) [reply rename-project.bash]: 👍 done

**Review by brightonsbox — APPROVED** [review]
LGTM - checked the script renames, and that the application boots up and displays everything properly 👍

**Pull request merged**